lumicks/pylake/kymotracker/detail/gaussian_mle.py

```diff
@@ -179,6 +179,12 @@ def gaussian_mle_1d(
         num_peaks=num_peaks,
     )
 
+    # We want to preserve the original order, so we sort, so that we can also sort afterwards
+    # and then apply the inverse sort.
+    idx = np.argsort(initial_position)
+    initial_position = initial_position[idx]
+    reverse_idx = np.argsort(idx)
```

Comment:
I find this comment confusing. Maybe break it up into 2 distinct thoughts/lines:
```
# Sort the positions for consistency during optimization
# Keep a set of reverse sort indices so the original order can be preserved on output
```

Reply:
Good point. Done!